tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LongLivedTokenGetContract.scala

```diff
@@ -0,0 +1,325 @@
+
+import io.netty.handler.codec.http.HttpHeaderNames.ACCEPT
+import io.restassured.RestAssured.{`given`, requestSpecification}
+import io.restassured.http.ContentType.JSON
+import net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson
+import org.apache.http.HttpStatus
+import org.apache.james.GuiceJamesServer
+import org.apache.james.jmap.core.ResponseObject.SESSION_STATE
+import org.apache.james.jmap.core.UuidState.INSTANCE
+import org.apache.james.jmap.http.UserCredential
+import org.apache.james.jmap.rfc8621.contract.Fixture.{ACCEPT_RFC8621_VERSION_HEADER, ACCOUNT_ID, BOB, BOB_PASSWORD, DOMAIN, authScheme, baseRequestSpecBuilder}
+import org.apache.james.utils.DataProbeImpl
+import org.junit.jupiter.api.{BeforeEach, Test}
+
+trait LongLivedTokenGetContract {
+  @BeforeEach
+  def setUp(server: GuiceJamesServer): Unit = {
+    server.getProbe(classOf[DataProbeImpl])
+      .fluent()
+      .addDomain(DOMAIN.asString())
+      .addUser(BOB.asString(), BOB_PASSWORD)
+
+    requestSpecification = baseRequestSpecBuilder(server)
+      .setAuth(authScheme(UserCredential(BOB, BOB_PASSWORD)))
+      .build
+  }
+
+  @Test
+  def getAllShouldSucceed(): Unit = {
+    val request: String =
+      s"""{
+         |  "using": ["urn:ietf:params:jmap:core", "com:linagora:params:long:lived:token"],
+         |  "methodCalls": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "create": {
+         |        "K38": {
+         |          "deviceId": "My android device"
+         |        },
+         |        "K39": {
+         |          "deviceId": "My IOS device"
+         |        }
+         |      }
+         |    }, "c1"],
+         |    ["LongLivedToken/get", {
+         |        "accountId": "$ACCOUNT_ID",
+         |        "ids": null
+         |      }, "c2"]
+         |  ]
+         |}""".stripMargin
+
+    val response = `given`()
+      .header(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
+      .body(request)
+    .when()
+      .post()
+    .`then`
+      .log().ifValidationFails()
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    assertThatJson(response)
+      .isEqualTo(
+      s"""{
+         |  "sessionState": "${SESSION_STATE.value}",
+         |  "methodResponses": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "created": {
+         |        "K38": {
+         |          "id": "$${json-unit.ignore}",
+         |          "token": "$${json-unit.ignore}"
+         |        },
+         |        "K39": {
+         |          "id": "$${json-unit.ignore}",
+         |          "token": "$${json-unit.ignore}"
+         |        }
+         |      }
+         |    }, "c1"],
+         |    ["LongLivedToken/get", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "state": "${INSTANCE.value}",
+         |      "list": [
+         |        {
```

Comment:
should be ignored order list when compare json.
I had a fail test in pass https://github.com/linagora/tmail-backend/pull/237#issue-1004149519